ModsenOnlineStore.EmailAuthentication.API/Controllers/EmailAuthenticationController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ModsenOnlineStore.EmailAuthentication.Application.Interfaces;
+using System.Net.Mail;
+
+namespace ModsenOnlineStore.EmailAuthentication.API.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class EmailAuthenticationController : ControllerBase
+    {
+        private IEmailSendingService emailSendingService;
+        private IVerificationCodeGeneratior codeGeneratior;
```

Comment:
use private readonly for fields
add empty lines between fields, methods etc

Reply:
fixed